Jenkinsfile

```diff
@@ -115,24 +115,11 @@ fhBuildNode([labels: ['nodejs6-ubuntu']]) {
         ]
         fhcapComponentUpdate(updateParams)
         fhOpenshiftTemplatesComponentUpdate(updateParams)
```

Comment:
@vchepeli these 'fhOpenshiftTemplatesComponentUpdate(updateParams)' lines should be removed too I think.